source/deployment/guest-accounts.rst

```diff
@@ -0,0 +1,128 @@
+.. _guest-accounts:
+
+Guest Accounts (Beta) (E10 and E20)
+===================================
+
+Available in `Enterprise Edition E10 and higher <https://about.mattermost.com/pricing/>`__. 
+
+Guest Accounts are a new role within Mattermost Enterprise editions which allow organizations to collaborate with individuals (such as vendors and contractors) outside of their organization. The Guest Account feature controls access to channels and other members within the organization. 
+
+An organization may use guest accounts to work on a website project with resources from external design firm.  Guest accounts could also be used to collaborate with customers on a support issue.
+
+Guests can: 
+
+- Pin messages to channels
+- Use slash commands (with the exception of those used to invite members)
+- Favorite channels
+- Mute channels
+- Update their account settings
+
+Guests cannot: 
+
+- Discover public channels
+- Join open teams
+- Create direct messages or group messages with members who aren’t within the same channel
+
+Enabling Guest Accounts
+-----------------------------------
+To enable Guests Accounts: 
+
+1. Navigate to **System Console > Authentication > Guest Access**.
+2. Update **Enable Guest Access** to ”True”.
+3. **Whitelist domains that are acceptable for Guest Access** (Optional).  
+ - This allows the System Administrators to set a list of approved guest domains. If you have team domain restrictions, you will also need to add your guest domain to **Team Settings > Allow only users with a specific email domain to join this team**.
+4. **Enforce Multi-Factor Authentication (MFA) for your Guests** (Optional)
+ - If you are enforcing MFA for your users, you can optionally choose to enforce MFA for your guest users. 
+
+Guest Authentication
+-----------------------------
+Guests can access the Mattermost server via email authentication. Please ensure you enable email invites from **System Console > Signup > Enable Email Invitations**.   
+
+SAML 2.0 and AD/LDAP authentication will be provided in a future release.
+
+Inviting Guests to the Mattermost Server
+------------------------------------------------------
+Guests can be invited into one for more Mattermost channels within a team by System Administrators and roles that have the **Invite Guest** permission. 
```

Comment:
and these channels can be across multiple teams?

Reply:
updated